src/AdminConsole/Components/Shared/Cards/LinkCard.razor

```diff
@@ -0,0 +1,29 @@
+<a href="@Model.Link" target="_blank">
+    <div class="relative flex rounded-lg border bg-white p-4 shadow-sm focus:outline-none">
+        <div class="flex-auto flex flex-col space-y-1">
+            <span class="block text-sm font-medium text-gray-900">@Model.Title</span>
+            <span class="flex items-center text-sm text-gray-500">@Model.Description</span>
+        </div>
+        
+        @if (Model.Image != null)
+        {
+            <DynamicComponent Type="@Model.Image" Parameters="@(new Dictionary<string, object> { { "class", "flex-none w-10 h-10" } })"/>
```

Comment:
This just passes the type of the component, and allows us to inject any component dynamically. This is quite nice.